oneview_redfish_toolkit/app.py

```diff
@@ -369,8 +373,38 @@ def hp_oneview_client_exception(exception):
                 "the config file. SSLCertFile: {}, SSLKeyFile: {}.".
                 format(ssl_cert_file, ssl_key_file))
 
-        ssl_context = (ssl_cert_file, ssl_key_file)
-        app.run(host=host, port=port, debug=debug, ssl_context=ssl_context)
+        if is_dev_env:
+            ssl_context = (ssl_cert_file, ssl_key_file)
+            app.run(host=host, port=port, debug=debug, ssl_context=ssl_context)
+        else:
+            start_cherrypy(app,
+                           host=host,
+                           port=port,
+                           ssl_cert_file=ssl_cert_file,
+                           ssl_key_file=ssl_key_file)
+
+
+def start_cherrypy(app,
+                   host=None,
+                   port=None,
+                   ssl_cert_file=None,
+                   ssl_key_file=None):
+    cherrypy.config.update({
+        'environment': 'production',
+        'log.screen': False,
+        'server.socket_port': port,
+        'server.socket_host': host,
+        'server.ssl_certificate': ssl_cert_file,
+        'server.ssl_private_key': ssl_key_file
+    })
+
+    app_logged = TransLogger(app.wsgi_app, setup_console_handler=False)
+    cherrypy.tree.graft(app_logged, '/')
+
+    Daemonizer(cherrypy.engine).subscribe()
```

Comment:
I removed the code that creates pid file because is not so important, I had added it before to make it easier to recover the process PID, but with the new code we would gain more complexity too.